Add proper Picture in Picture support for Android/iOS (#52)
bitmovin/bitmovin-player-react-native: README.md

    - [Setup Android Player SDK](#setup-android-player-sdk)
  - [Getting Started](#getting-started)
    - [Setting up a license key](#setting-up-a-license-key)
      - [Through code](#through-code)
      - [Through `Info.plist`](#through-infoplist)
      - [Through `AndroidManifest.xml`](#through-androidmanifestxml)
    - [Setting up the playback configuration](#setting-up-the-playback-configuration)
    - [Accessing native `Player` instances](#accessing-native-player-instances)
    - [Listening to events](#listening-to-events)
    - [Enabling DRM protection](#enabling-drm-protection)
      - [Prepare hooks](#prepare-hooks)
    - [Adding external subtitle tracks](#adding-external-subtitle-tracks)

>     - [Enabling picture-in-picture mode](#enabling-picture-in-picture-mode)

Author: Done 5d572ac1a2a5b82c39a57593aff5ffd62689bdeb

    android:supportsPictureInPicture="true"
    android:configChanges=
        "screenSize|smallestScreenSize|screenLayout|orientation"
    ...
```

#### iOS

**Set background modes capability**

Make sure to add the `UIBackgroundModes` key to the `dict` section of your `Info.plist`:

```xml
<key>UIBackgroundModes</key>
<array>
  <string>audio</string>
</array>
```

This step can also be performed from [Xcode](https://developer.apple.com/documentation/xcode/configuring-background-execution-modes).

**Configure audio session on app startup**

Configure your app's `AudioSession` category to `playback` during the main component's initialization:

```typescript
import { AudioSession } from 'bitmovin-player-react-native';

// App's root component
const App = () => {
  useEffect(() => {
    // Set your app's `AudioSession` category to `playback` on initialization.
    // Please, note even though this step is required for iOS it won't take any effect on Android.
    AudioSession.setCategory('playback').catch((error) => {
      // Handle any native error that might occur during this process.
      handleError(error);
    });
  });
  // ...
  return /* ... */;
};
```

> This is step is required in order to properly enable background reproduction of audio content on iOS. Without it, the picture-in-picture option appears on the player UI but doesn't work.

Reviewer: ```suggestion
This is step is required in order to properly enable background playback on iOS. Without it, the Picture in Picture option appears on the player UI but has no effect when used.
```

Author: Done 5d572ac1a2a5b82c39a57593aff5ffd62689bdeb
